wolfssl/wolfcrypt/settings.h

```diff
@@ -212,6 +212,12 @@
 /* Uncomment next line if using RENESAS RX64N */
 /* #define WOLFSSL_RENESAS_RX65N */
 
+/* Uncomment next line if using RENESAS SCE PROTECT MODE */
```

Comment:
Should our comment say "Renesas SCE Protected Mode" instead of "PROTECT MODE"?  Seems like Renesas uses the term "Protected" instead of "PROTECT".

Reply:
changed